MOBILE_CLIENT/COMMON/_HIGH/INSECURE_STORAGE/description.md

```diff
@@ -0,0 +1,145 @@
+Insecure storage vulnerability in mobile applications refers to a security flaw where sensitive data, such as user credentials, personal information, or other confidential data, is stored on the device in world readable locations. This can leave the data vulnerable to unauthorized access by other applications on the device or by a malicious actor with physical access to the device.
```

Comment:
If you tried this in preview it's not working can you please verify it

Reply:
Good catch! Had a format issue in code examples